docs/development/codeguide.rst

```diff
@@ -148,6 +150,11 @@ Coding Style/Conventions
       * or one can add a single ``# fmt: skip`` comment to the end of a line to
         disable ``black`` formatting for that line.
 
+      .. note::
+        ``black`` can also be manipulated into one-lining (assuming the statement
+        fits inside the 88 character line length limit) or multi-lining a statement
+        by adding a trailing comma to the end of the line.
```

Comment:
I know the use case this refers to, but I am not sure I'd understand it from the description (actually I recall it is enabling more "matrix-like" formatting, but what exactly is the effect – allowing > 1 item per line in a multi-line block?).
I think @eteq's comment on [adding a `+`](https://astropy.slack.com/archives/C04A0DV3680/p1668201774735609) might also be useful here.

Reply:
Maybe something like this:
```
You can communicate to ``black`` that a list or array should be formatted as one item
per line using the `magic trailing comma 
<https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#the-magic-trailing-comma>`_.
This frequently helps with formatting arrays.
```